verilog/tools/ls/autoexpand_test.cc

```diff
@@ -20,50 +20,88 @@
 
 namespace verilog {
 namespace {
+using verible::lsp::CodeAction;
 using verible::lsp::CodeActionParams;
 using verible::lsp::EditTextBuffer;
 using verible::lsp::Range;
 using verible::lsp::TextDocumentContentChangeEvent;
 using verible::lsp::TextEdit;
 
-void TestTextEdits(
-    const std::function<std::vector<TextEdit>(BufferTracker*)>& editFun,
-    absl::string_view textBefore, absl::string_view textGolden,
-    bool repeat = true) {
-  EditTextBuffer buffer(textBefore);
+// Generate text edits using the given function and test if they had the desired
+// effect
+void TestTextEdits(const std::function<std::vector<TextEdit>(
+                       SymbolTableHandler*, BufferTracker*)>& edit_fun,
+                   const std::vector<absl::string_view>& project_file_contents,
+                   const absl::string_view text_before,
+                   const absl::string_view text_golden,
+                   const bool repeat = true) {
+  static const char* TESTED_FILENAME = "<<tested-file>>";
+  // Init a text buffer which we need for the autoepxand functions
+  EditTextBuffer buffer(text_before);
   BufferTracker tracker;
-  tracker.Update("<<test>>", buffer);
-  std::vector<TextEdit> edits = editFun(&tracker);
+  tracker.Update(TESTED_FILENAME, buffer);
+  // Create a Verilog project with the given project file contents
+  const std::shared_ptr<VerilogProject> proj =
+      std::make_shared<VerilogProject>(".", std::vector<std::string>());
+  size_t i = 0;
+  for (const absl::string_view file_contents : project_file_contents) {
+    auto filename = absl::StrCat("<<project-file-", std::to_string(i), ">>");
```

Comment:
You don't need `std::to_string()` when using StrCat() - it already does all necessary conversion.